edgelet/iotedge/src/init/import/old_config/mod.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+mod agent;
+pub(crate) use agent::*;
```

Comment:
(unrelated to the PR at hand)

Why use `pub(crate)` instead of just `pub` when working with `bin` crates? IMO it just makes things more verbose for no great reason, since there aren't any upstream crates to "hide" these details from.

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=048936fced270e6262c38750d9fcc86b vs https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=c9416b3a5cf256e6f40c254d44d805a0

Reply:
Ah, sure, that's actually pretty reasonable actually. Though I'm pretty sure you can work around that by marking top-level modules as `pub(crate)` and calling it a day, but sure.